### PR TITLE
Added dotnet-week1 homework

### PR DESCRIPTION
**Pull request by shruthidasarapu**

### CODE REVIEW

File: week1/task2/Program.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿// See https://aka.ms/new-console-template for more information
+string input = "Intellectualization";
+char[] vowels = new char[] { 'a', 'e', 'i', 'o', 'u' };
+int GetVowelCount(string text)
+{
+    return text.ToLower().Count(x => vowels.Contains(x));
+}
+int vowelCount = GetVowelCount(input); //TODO: Implement GetVowelCount 
+Console.WriteLine($"Number of vowels: {vowelCount}");
```
Comment (ElhamDargi) [inline week1/task2/Program.cs:9]: I was expecting people to try to solve this with a foreach, If, switch case, but you found a better solution with LINQ. 
Well done.

File: week1/task3/Program.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+{
+    for (int i = 0; i < arr.Length; i++)
+    {
+        if (arr[i] < 0)
+        {
+            sumOfNegativeNumbers += arr[i];
+        }
+        else if (arr[i] >= 0)
+        {
+            multipleOfPositiveNumbers = multipleOfPositiveNumbers * arr[i];
```
Comment (ElhamDargi) [inline week1/task3/Program.cs]: Very good!!
You can also write this line like this:
multipleOfPositiveNumbers  *= arr[i];

File: week1/task4/Program.cs [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿// See https://aka.ms/new-console-template for more information
+int n = 6;
+int Fibonacci(int n)
+{
+    if ((n == 0) || (n == 1))
+    {
+        return n;
+    }
+    else
+        return Fibonacci(n - 1) + Fibonacci(n - 2);
```
Comment (ElhamDargi) [inline week1/task4/Program.cs:10]: Excellent!!!!
You used a recursive version.

File: week1/task3/Program.cs (hunk 2 above)
Comment (shruthidasarapu) [reply week1/task3/Program.cs]: yes, I will replace it.

**Pull request merged**